src/coordinax/_src/base/__init__.py

```diff
@@ -4,18 +4,18 @@
     # Base
     "AbstractVector",
     "ToUnitsOptions",
-    # Position
-    "AbstractPosition",
-    # Velocity
-    "AbstractVelocity",
-    # Acceleration
-    "AbstractAcceleration",
+    # Pos
+    "AbstractPos",
+    # Vel
+    "AbstractVel",
+    # Acc
```

Comment:
Total nit pick, but comments should probably stay as full words Position, Velocity, Acceleration.

Reply:
Agreed. Product of global find & replace